pom.xml

```diff
@@ -199,13 +191,13 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>metrics</artifactId>
-            <version>3.1.2.9</version>
+            <version>4.0.2.10-SNAPSHOT</version>
```

Comment:
we'll have to wait to merge this until the metrics plugin PR gets merged upstream.